kernel/packages/shared/meta/types.ts

```diff
@@ -22,20 +22,21 @@ export type MetaConfiguration = {
 export type WorldConfig = {
   pois: Vector2Component[]
   renderProfile?: RenderProfile
-  messageOfTheDay?: MessageOfTheDayConfig
+  messageOfTheDay?: MessageOfTheDayConfig | null
+  messageOfTheDayInit?: boolean
 }
 
 export type MessageOfTheDayConfig = {
-  imageUrl: string
-  endUnixTimestamp: number
+  background_banner: string
+  endUnixTimestamp?: number
```

Comment:
We are going to implement the timestamp logic?

Reply:
The endpoint doesn't provide any timestamp, we are leaving the field as optional for the future but it will likely go away.
Since the endpoint is customized by wallet I assume it only returns something when a MOTD is set.

Reply:
Ok! thanks for the clarification.